resources/assets/js/features/trip/components/Forms/TripForm.js

```diff
@@ -195,6 +213,47 @@ class TripForm extends React.Component {
                        onWaypointDelete={onWaypointDelete}
                     />
 
+                    {!tripData.id &&
+                        <div className="form-group row">
+                            <label className="form-control-label text-muted col-sm-4" htmlFor="is_in_both_directions">
+                                {translate('trip_form.is_round_trip')}
+                            </label>
+                            <div className="col-sm-8">
+                                <div className="form-check">
+                                    <label className="form-check-label">
+                                        <input className="form-check-input" type="checkbox" id="is_in_both_directions"
+                                               checked={this.state.isInBothDirections}
+                                               name="is_in_both_directions"
+                                               onChange={this.toggleInBothDirections.bind(this)}
+                                        />
+                                    </label>
+                                </div>
+                            </div>
+                        </div>
+                    }
+
+                    {this.state.isInBothDirections &&
```

Comment:
avoid such statements in template, move this part to separate method, that will return or not html based on state